plugins/wporg-5ftf/includes/pledge-form.php

```diff
@@ -95,11 +95,11 @@ function process_form_new() {
 	Contributor\add_pledge_contributors( $new_pledge_id, $contributors );
 
 	// Attach logo to the pledge.
-	wp_update_post( array(
+	/*wp_update_post( array(
 		'ID'          => $logo_attachment_id,
 		'post_parent' => $new_pledge_id,
 	) );
-	set_post_thumbnail( $new_pledge_id, $logo_attachment_id );
+	set_post_thumbnail( $new_pledge_id, $logo_attachment_id );*/
```

Comment:
@coreymckrill I'm guessing we'll need to reactivate this for other pledges to be submitted today, is that right?

Reply:
Ah crap.

Reply:
Good catch

Reply:
Fixed in 454962522db7b8355c95d048496f1eeeab2c478f